cmd/fpga_crihook/main.go

```diff
@@ -176,7 +174,7 @@ func (he *hookEnv) getFPGAParams(config *Config) ([]fpgaParams, error) {
 			if dev.processed {
 				continue
 			}
-			port, err := fpga.NewPort(deviceName)
+			port, err := newPort(deviceName)
```

Comment:
You can also call `he.newPort(deviceName)` here. No need to change the signature of `getFPGAParams()`.

Reply:
thanks. done